src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -98,4 +104,24 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
             return $data;
         };
     }
+
+    /**
+     * @param object $item
+     *
+     * @throws Error
+     */
+    private function validate($item, ResolveInfo $info, ResourceMetadata $resourceMetadata, string $operationName = null)
+    {
+        if (null === $this->validator) {
+            return;
```

Comment:
We don't want to warn the user if there is no validator?

Reply:
No (we would just make this property not nullable otherwise), it allows to create micro-frameworks on top of API Platform and it mimics the behavior of the `ValidateListener` (which is optional).